packages/core/src/lib/file-info/file-info.ts

```diff
@@ -19,7 +19,7 @@ export default class FileInfo {
   }
 }
 
-type NestedArray = Array<string | NestedArray>;
+type NestedArray = (string | NestedArray)[];
```

Comment:
What is the a reason why we go here with `Array`?

Reply:
EsLint has thrown an error